lib/config/config.dart

```diff
@@ -174,7 +174,7 @@ class Config {
   final MacOSConfig? macOSConfig;
 
   /// Creates [Config] icons from [json]
-  factory Config.fromJson(Map json) => _$ConfigFromJson(json);
+  factory Config.fromJson(Map<dynamic, dynamic> json) => _$ConfigFromJson(json);
```

Comment:
```suggestion
  factory Config.fromJson(Map<String, dynamic> json) => _$ConfigFromJson(json);
```

Reply:
I forgot to mention that this was omitted specifically while defining type because `YamlMap` cannot be casted to `Map<String, dynamic>` but can be to `Map<dynamic, dynamic>`

Reply:
Leave a comment above the line so we don't forget and it means everyone else will be aware that `YamlMap` cannot be casted to `Map<String, dynamic>`

Reply:
@MarkOSullivan94 nice suggestion. Done ✅